pages/admin/lessons/dojo/modules.tsx

```diff
@@ -0,0 +1,126 @@
+import { gql, useQuery } from '@apollo/client'
```

Comment:
path should be: `/pages/admin/lessons/:lessonId/modules`

Reply:
We don't need a `dojo` in the url path

Reply:
Removing the rest of breadcrumb items will be done in a different PR